3_NConVert.cpp

```diff
@@ -43,11 +43,46 @@ string convert(string s, int numRows) {
 	return str;
 }
 
+// ����N�͹��ɣ�ֱ�Ӷ�λ�±�
```

Comment:
It seems the wrong encoding. MS loves GBK for Chinese Characters, while many online environments use UTF-8.
Try to make it consistent.